_includes/tutorials/fk-joins/kstreams/code/tutorial-steps/dev/wait-for-containers.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function readiness_probe {
+    nc -z -w 2 0.0.0.0 29092
+}
+
+echo "Waiting for the broker to become available ..."
+
+readiness_probe
+
+while [[ $? != 0 ]]; do
+    sleep 5
+    readiness_probe
+done
```

Comment:
There is a built in bash function (until) for this if you're interested...

```
until readiness_probe
do
  sleep 5
done

Reply:
thanks but I used pre-existing a `wait-for-containers.sh` file.  It seems most/all of the streams examples use the same format.  IMHO if we make that change, we can do a follow-up PR to change all files to use the same format.